### PR TITLE
[WIP] Markets page should update list of orders when order is created

### PR DESCRIPTION
**Pull request by amir-budaychiev**
[WIP] Markets page should update list of orders when order is created

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -70,6 +70,7 @@ gem 'arel-is-blank', '~> 1.0'
 gem 'sentry-raven', '~> 2.7', require: false
 
 group :development, :test do
+  gem 'spirit_hands'
```
Comment (yivo) [inline Gemfile]: @amir-budaychiev Can you please remove this? We use `pry-byebug` instead.

File: app/controllers/concerns/order_creation.rb [hunk 2]
```diff
@@ -11,17 +11,18 @@ def order_params(order)
...
-        :state, :origin_volume, :volume, :member_id, :ord_type)
+          :bid, :ask, :currency, :price, :source,
+          :state, :origin_volume, :volume, :member_id, :ord_type)
     end
 
     def order_submit
       begin
         Ordering.new(@order).submit
         render status: 200, json: success_result
-      rescue => e
```
Comment (yivo) [inline app/controllers/concerns/order_creation.rb:22]: @amir-budaychiev Please change base branch to devel and rebase. We have this problem fixed.

**Review by yivo — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (amir-budaychiev) [reply Gemfile]: @yivo done

File: app/controllers/concerns/order_creation.rb (hunk 2 above)
Comment (amir-budaychiev) [reply app/controllers/concerns/order_creation.rb]: @yivo Do I need create new PULL REQUEST into rubykube:devel from amir-budaychiev:devel?

Comment (amir-budaychiev) [reply app/controllers/concerns/order_creation.rb:22]: @yivo I did it for devel branch in the new PR: https://github.com/rubykube/peatio/pull/455. Can we close that issue?

Comment (yivo) [reply app/controllers/concerns/order_creation.rb:22]: @amir-budaychiev Yes, sure. I will close it.

**Comment by yivo** [comment]
Moved to #455.

**Pull request closed without merging**